fabtests/common/shared.c

```diff
@@ -3784,3 +3789,30 @@ int ft_parse_long_opts(int op, char *optarg)
 		return EXIT_FAILURE;
 	}
 }
+
+int ft_sync_for_validation()
```

Comment:
I have discovered that this function is causing stack corruption.  I'll need to fix it if we decide to continue working this PR.